db/migrate/20231205140630_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postsCounter, default: 0
```

Comment:
- Hey, the naming convention in ruby uses the snake_case, kindly update the postsCounter to posts_counter.